src/test/java/teammates/test/cases/ui/browsertests/InstructorHomePageUiTest.java

```diff
@@ -396,7 +396,7 @@ public void testCopyToFsAction() throws Exception {
         homePage.fillCopyToOtherCoursesForm("New name!");
         
         homePage.clickFsCopySubmitButton();
-        
+        homePage.waitForPageToLoad();
```

Comment:
Do the wait inside clickFsCopySubmitButton? Then there will be no need for developers to remember it every time clickFsCopySubmitButton is used.